getting-started/python/index.md

```diff
@@ -46,42 +44,42 @@ is your central gateway to
 
 ### Collections
 
-The EO data available at a back-end is organised in so-called collections.
-For example, a back-end might provide fundamental satellite collections like "Sentinel 1" or "Sentinel 2",
+The Earth observation data (the input of your openEO jobs) is organised in so-called collections,
+e.g. fundamental satellite collections like "Sentinel 1" or "Sentinel 2",
 or preprocessed collections like "NDVI".
-Collections are used as input data for your openEO jobs.
 
 ::: tip Note
 More information on how openEO "collections" relate to terminology used in other systems can be found in
 [the openEO glossary](https://openeo.org/documentation/1.0/glossary.html#eo-data-collections).
 :::
 
 
-Let's list all available collections on the back-end,
-using [`list_collections`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.list_collections):
+While it's recommended to browse the available EO collections on the  
+[openEO Platform collections overview webpage](../../data-collections/index.md),
+it's possible to list and inspect them programmatically, 
+as very simple example of openEO Python client usage,
+using methods on the `connection` object we just created 
+(like [`list_collection_ids`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.list_collection_ids))
+or [`describe_collection`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.describe_collection)):
 
-```python
-print(connection.list_collections())
 ```
+>>> # Get all collection ids
+>>> print(connection.list_collection_ids())
+['AGERA5', 'SENTINEL1_GRD', 'SENTINEL2_L2A', ...
 
-which returns list of collection metadata dictionaries, e.g. something like:
+>>> # Get a listing of all collections with metadata
+>>> print(connection.list_collections())
+[{'id': 'AGERA5', 'title': 'ECMWF AGERA5 meteo dataset ...', 'description': ...
 
-```
-[{'id': 'AGERA5', 'title': 'ECMWF AGERA5 meteo dataset', 'description': 'Daily surface meteorolociga datal ...', ...},
- {'id': 'SENTINEL2_L2A_SENTINELHUB', 'title': 'Sentinel-2 top of canopy', ...},
- {'id': 'SENTINEL1_GRD', ...},
- ...]
+>>> # Get metadata of a single collection
+>>> print(connection.describe_collection("SENTINEL2_L2A"))
+{'id': 'SENTINEL2_L2A', 'title': 'Sentinel-2 top of canopy ...', 'stac_version': '0.9.0', ...
 ```
 
-This listing includes basic metadata for each collection.
```

Comment:
This paragraph I think is important to mention, because many people don't realize that list_collections is not complete.

Reply:
I don't know, I don't think this level of collection metadata details are useful for a "getting started" phase. 
Inspecting collection metadata is a lot more practical in a web editor or web component than to do programmatically, so I wouldn't spend too much time on it

Reply:
If the Python client docs would highlight this issue, I'd be okay with not mentioning it here. But as far as I can see, the Python client docs don't mention it so I'd at least mention it here (or update the Python client docs).

Reply:
What exactly do you refer to?
That the python docs, e.g. [Getting Started](https://open-eo.github.io/openeo-python-client/basics.html) should mention `connection.list_collections()` ?
Or that the documentation of `list_collection()` should mention that the collection metadata has less fields than per-collection metadata from `describe_collection(cid)`?

Reply:
As you are linking to https://open-eo.github.io/openeo-python-client/data_access.html below, I'd add it there and in the documentation of list_collection.

Reply:
For [the ](https://open-eo.github.io/openeo-python-client/data_access.html) something like:

> [list_collections()](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.list_collections) to list basic metadata for all collections
> [describe_collection()](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.describe_collection) to show the complete metadata for a particular collection

Reply:
Good point, those bits of documentation desperately needed some polishing. The "imagecollections types" stuff is apparently some old doc leftovers from 2018 :smile: 

Should be improved now

Reply:
Looks good to me, thanks!